### ISSUE TITLE
Settings->Main: the `quickAction` function should be called on form submit, not on button click

### ISSUE THREAD

**Issue by mesosoi** [mesosoi/silverwaregames-io-issue-tracker#149]
Settings->Main: the `quickAction` function should be called on form submit, not on button click, so that it would be possible to use forms with keyboard only.

**Comment by mesosoi** [comment]
Let's get this done quick because our site must have solid accessibility
